exchanges/gateio/ws_ob_update_manager.go

```diff
@@ -99,10 +100,26 @@ func (m *wsOBUpdateManager) LoadCache(p currency.Pair, a asset.Item) *updateCach
 
 // SyncOrderbook fetches and synchronises an orderbook snapshot to the limit size so that pending updates can be
 // applied to the orderbook.
-func (c *updateCache) SyncOrderbook(ctx context.Context, g *Gateio, pair currency.Pair, a asset.Item, limit uint64) error {
-	// TODO: When templates are introduced for all assets define channel key and use g.Websocket.GetSubscription(ChannelKey{&subscription.Subscription{Channel: channelName}})
-	// to get the subscription and levels for limit. So that this can scale to config changes. Spot is currently the only
-	// asset with templates but it has only one level.
+func (c *updateCache) SyncOrderbook(ctx context.Context, g *Gateio, pair currency.Pair, a asset.Item) error {
+	// TODO: When templates are introduced for all assets, scale to dynamic supported limit/levels provided from config.
```

Comment:
🐒 The word dynamic kinda feels wrong when it's actually "configured".
Dynamic normally means more variable than that.
How's this feel:
```suggestion
	// TODO: When subscription config is added for all assets update limits to use sub.Levels
```